### PR TITLE
corrected batch norm implementation in cntk and documentation in tensorflow

### PR DESCRIPTION
**Pull request by sachinruk**
I corrected the documentation in tensorflow backend and the implementation in cntk backend, as to how the epsilon is used. This relates to this question: https://stackoverflow.com/questions/50830891/recreating-keras-model-in-numpy

Basically BN was used as (x-mean)/(sqrt(var) + eps) whereas it should have been used as  (x-mean)/sqrt(var + eps).

Also this line seems to confirm that its used the way I have described above:
https://github.com/tensorflow/tensorflow/blob/r1.8/tensorflow/python/ops/nn_impl.py#L830

### CODE REVIEW

**Comment by taehoonlee** [comment]
@fchollet, This is very important correction. Most models on CNTK have not been agreed with TF/TH (keras-team/keras-applications#12), but now all the models can produce the same results over all the backends. Thank you for the correction @sachinruk.
